### PR TITLE
#160094811 Setup ESLint

### PR DESCRIPTION
**Pull request by lytes20**
#### What does this PR do?
- Setup Eslint
#### Description of Task to be completed?
- Configure Eslint for identifying and reporting on patterns in JavaScript
#### How should this be manually tested?
- By writting react javascript code that does not follow `AirBnb` style guidelines
#### Any background context you want to provide?
- Setup `ESLint`, set up `Prettier`. Prettier formats the code based on ESLint rules. And the ESLint rules are those of AirBnb
#### What are the relevant pivotal tracker stories?
[#160094811](https://www.pivotaltracker.com/story/show/160094811)
#### Screenshots (if appropriate)
Lint errors show on trying to run the app
screen shot 2018-08-29 at 18 07 51

### CODE REVIEW

**Review by RutaleIvanPaul — COMMENTED** [review]
Any effect of setting up  ESlint to the Prettier extension?  https://www.youtube.com/watch?v=YIvjKId9m2c seems to suggest there could be conflicts

**Comment by lytes20** [comment]
Hey, @RutaleIvanPaul the video u sent explains how you can resolve an issue of having both Prettier and ESLint extensions set up and enabled in your VScode, If both of them are enabled for your VScode, they will both try to format your code. So you only need one installed.
Prettier is, however, recommended to format the code. So u can just use ESLint to enforce the guidelines. Prettier is only a code "formatter" and can not highlight when you don't follow guidelines.
Also, prettier is code rules suggest double quotes by default but i override this rule as seen in this screenshot below.
screen shot 2018-08-29 at 19 41 48
There are basically no conflicts in this setup as regards Prettier and ESLint.
You just need to make sure you also avoid them locally in your VScode configurations

**Review by AGMETEOR — APPROVED** [review]

**Review by abulojoshua1 — APPROVED** [review]

**Review by phillipseryazi — APPROVED** [review]

**Review by RutaleIvanPaul — APPROVED** [review]

**Review by ssewilliam — APPROVED** [review]

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["airbnb", "prettier"],
+  "plugins": ["prettier"],
```
Comment (geofrocker) [inline .eslintrc.json]: Any reason why you are not using the react plugin

**Review by geofrocker — CHANGES_REQUESTED** [review]

**Comment by lytes20** [comment]
@geofrocker, i have included the react plugin

**Pull request merged**